.gitignore

```diff
@@ -0,0 +1,2 @@
+/test.md
+/.github/workflows/linters.yml
```

Comment:
- [ ] You did a good effort for changes provided by the previous review but there is a few changes you still need to work on for you to move to another milestone, As required for this project you should ignore the `node_modeles` and `test.md` file in `.gitignore` but I noticed that you ignore the workflows linters.yml which are not supposed to be in this file 

**Your .gitignore should look like this** :point_down: 
![gitignore](https://user-images.githubusercontent.com/88894525/191436920-d69c63c6-9697-4c91-98b0-0b7a7bc0f50a.png)